duo-256.nix

```diff
@@ -28,6 +28,11 @@
 # patient.  The password is "milkv". Native ethernet is untested. GPIO is
 # untested.
 #
+# Currently the Duo will have no NAT access to the larger internet, it will
+# only be able to contact the host machine.  In the future it would be nice to
+# give it access.  So far I've been unsuccessful at that.  See
```

Comment:
I think networkd on the host side can setup NAT automatically. The option is `IPMasquerade`.